src/Sulu/Bundle/AdminBundle/Metadata/FormMetadata/OptionMetadata.php

```diff
@@ -32,7 +32,7 @@ class OptionMetadata
     protected $type;
 
     /**
-     * @var string|int|OptionMetadata[]
+     * @var bool|int|string|float|OptionMetadata[]|null|mixed
```

Comment:
`float` is not supported by our parser and is kept as string:

https://github.com/sulu/sulu/blob/512639aebb408774d726e38be715c5f9e851c97e/src/Sulu/Bundle/AdminBundle/Metadata/XmlParserTrait.php#L26

https://github.com/sulu/sulu/blob/512639aebb408774d726e38be715c5f9e851c97e/src/Sulu/Bundle/AdminBundle/Metadata/FormMetadata/Parser/PropertiesXmlParser.php#L391-L392

```suggestion
     * @var bool|int|string|null|OptionMetadata[]
```


We should not support mixed here.